packages/@aws-cdk/aws-events-targets/package.json

```diff
@@ -131,6 +131,7 @@
   "awslint": {
     "exclude": [
       "docs-public-apis:@aws-cdk/aws-events-targets.EcsTask.securityGroup",
+      "docs-public-apis:@aws-cdk/aws-events-targets.EcsTask.securityGroups",
```

Comment:
You shouldn't need to add this. The documentation above the construct property securityGroups should be sufficient.